src/lib/components/ui/input.svelte

```diff
@@ -346,6 +414,12 @@
 			font-size: var(--font-size-small);
 			line-height: var(--line-height-small);
 			letter-spacing: var(--letter-spacing-small);
+			&.search {
+				padding-left: 32px;
```

Comment:
It would be good to use variables for paddings and spacing so if the `diete.css` changes the components are automatically resized correctly. There is `--padding` which is `1rem` which is `16px`. There is also `--half-padding` and `--double-padding`.